src/ethereum_test_fixtures/blockchain.py

```diff
@@ -401,13 +403,29 @@ def from_withdrawal(cls, w: WithdrawalGeneric) -> "FixtureWithdrawal":
         return cls(**w.model_dump())
 
 
+@dataclass(slots=True)
+class WitnessChunk:
+    """Represents execution witness data for a block."""
+
+    state: List[str]
+    codes: List[str]
+    keys: List[str]
+    headers: List[str]
+
+    @classmethod
+    def from_json(cls, s: str) -> List["WitnessChunk"]:
+        """Parse witness chunks from JSON string."""
+        return [cls(**obj) for obj in json.loads(s)]
+
+
 class FixtureBlockBase(CamelModel):
     """Representation of an Ethereum block within a test Fixture without RLP bytes."""
 
     header: FixtureHeader = Field(..., alias="blockHeader")
     txs: List[FixtureTransaction] = Field(default_factory=list, alias="transactions")
     ommers: List[FixtureHeader] = Field(default_factory=list, alias="uncleHeaders")
     withdrawals: List[FixtureWithdrawal] | None = None
+    execution_witness: WitnessChunk | None = Field(None, alias="executionWitness")
```

Comment:
I don't know how is this not failing since `WitnessChunk` is not a pydantic model.

Reply:
I guess it's only because pydantic is an incredibly flexible library that correctly guesses the correct thing to do.